source/implementations/f7/Meadow.F7/Devices/Esp32Coprocessor/SharedEnums.cs

```diff
@@ -489,6 +489,37 @@ public enum CellFunction
         /// CellFunction - NetworkDisconnectedEvent
         /// </summary>
         NetworkDisconnectedEvent = 1,
+        /// <summary>
+        /// CellFunction - NetworkErrorEvent
+        /// </summary>
+        NetworkErrorEvent = 2,
+    };
+
+    /// <summary>
+    /// Cell error 
+    /// </summary>
+    public enum CellError
```

Comment:
Please, adapt the following error names according to the defined in the Meadow OS.